packages/st2mistral/conf/logrotate.conf

```diff
@@ -0,0 +1,25 @@
+## Mistral API
+/var/log/mistral/mistral-api.log {
+  size 1M
+  rotate 5
+  compress
+  notifempty
+  missingok
+  create 644 mistral mistral
+  postrotate
+    pkill -USR1 -f mistral.api.wsgi > /dev/null 2>&1 || true
+  endscript
+}
+
+## Mistral Server
+/var/log/mistral/mistral-server.log {
+  size 50M
+  rotate 10
```

Comment:
Only these actual logrotate settings are under question.
If it's good enough and balanced or we need different numbers here.